frontend/packages/dev-console/integration-tests/support/pages/app.ts

```diff
@@ -0,0 +1,202 @@
+            // cy.get('#tour-step-footer-primary').should('be.visible').click();
+          }
+        });
+        break;
+      }
+      case switchPerspective.Developer: {
+        cy.get('li[role="menuitem"]')
+          .contains('Developer')
+          .click();
+        cy.get('body div').then(($el) => {
+          if ($el.find('#guided-tour-modal').length !== 0) {
+            cy.get('#tour-step-footer-secondary').click();
+            // cy.get('#tour-step-footer-primary').should('be.visible').click();
+          }
+        });
+        break;
+      }
+      default: {
+        throw new Error('Option is not available');
+      }
+    }
+  },
+};
+
+export const naviagteTo = (opt: devNavigationMenu) => {
+  switch (opt) {
+    case devNavigationMenu.Add: {
+      cy.byLegacyTestID('+Add-header')
+        .click()
+        .then(() => {
+          cy.url().should('include', 'add');
+          app.waitForLoad();
+        });
+      break;
+    }
+    case devNavigationMenu.Topology: {
+      cy.byLegacyTestID('topology-header')
+        .click()
+        .then(() => {
+          cy.url().should('include', 'topology');
+        });
+      break;
+    }
+    case devNavigationMenu.GitOps: {
+      cy.byLegacyTestID('gitops-header')
+        .click()
+        .then(() => {
+          cy.pageTitleShouldContain('GitOps');
+        });
+      break;
+    }
+    case devNavigationMenu.Monitoring: {
+      cy.byLegacyTestID('monitoring-header')
+        .click()
+        .then(() => {
+          cy.pageTitleShouldContain('Monitoring');
+        });
+      break;
+    }
+    case devNavigationMenu.Builds: {
+      cy.byLegacyTestID('build-header').click();
+      cy.pageTitleShouldContain('Build Configs');
+      break;
+    }
+    case devNavigationMenu.Pipelines: {
+      cy.byLegacyTestID('pipeline-header').click();
+      cy.pageTitleShouldContain('Pipelines');
+      break;
+    }
+    case devNavigationMenu.Search: {
+      cy.byLegacyTestID('search-header').click();
+      cy.pageTitleShouldContain('Search');
+      break;
+    }
+    case devNavigationMenu.Helm: {
+      cy.byLegacyTestID('helm-releases-header').click();
+      cy.pageTitleShouldContain('Helm Releases');
+      break;
+    }
+    case devNavigationMenu.Project: {
+      cy.byLegacyTestID('project-details-header').click();
+      break;
+    }
+    case devNavigationMenu.ConfigMaps: {
+      cy.get('#ConfigMap').click();
+      cy.pageTitleShouldContain('Config Maps');
+      break;
+    }
+    case devNavigationMenu.Secrets: {
+      cy.get('#Secret').click();
+      cy.pageTitleShouldContain('Secrets');
+      break;
+    }
+    default: {
+      throw new Error('Option is not available');
+    }
+  }
+};
+
+export const projectNameSpace = {
+  selectCreateProjectOption: () => {
+    cy.byLegacyTestID('namespace-bar-dropdown')
+      .find('button')
+      .eq(0)
+      .click();
+    cy.byTestDropDownMenu('#CREATE_RESOURCE_ACTION#').click();
```

Comment:
Were is `#CREATE_RESOURCE_ACTION#` defined?  -thanks

Reply:
It's an id attribute from dom structure

Reply:
oh, ok, weird that it also ends in `#`.